en/deployment/articles/03-installing-a-search-engine/02-elasticsearch/07-security.markdown

```diff
@@ -169,27 +175,47 @@ configuration (`elasticsearch.yml`) used in this guide (with the Elasticsearch 6
 example commented out):
 
 ```yaml
-# For Elasticsearch 7
+# For Elasticsearch 7.3/7.4
 cluster.name: LiferayElasticsearchCluster
+
+# X-Pack Security
 xpack.security.enabled: true
+
+## TLS/SSL settings for Transport layer
 xpack.security.transport.ssl.enabled: true
-xpack.security.transport.ssl.verification_mode: certificate
-xpack.security.transport.ssl.key: /path/to/[Elasticsearch Home]/config/localhost.crt
-xpack.security.transport.ssl.certificate: /path/to/[Elasticsearch Home]/config/localhost.key
-xpack.security.transport.ssl.certificate_authorities: ["/path/to/ca.crt"]
+xpack.security.transport.ssl.verification_mode: certificate 
+xpack.security.transport.ssl.key: /path/to/localhost.key
+xpack.security.transport.ssl.certificate: /path/to/localhost.crt
+xpack.security.transport.ssl.certificate_authorities : ["/path/to/ca.crt"]
+
+# TLS/SSL settings for HTTP layer
 xpack.security.http.ssl.enabled: true
-xpack.security.http.ssl.certificate: /path/to/[Elasticsearch Home]/config/localhost.crt
-xpack.security.http.ssl.key: /path/to/[Elasticsearch Home]/config/localhost.key
-xpack.security.http.ssl.certificate_authorities: ["/path/to/ca.crt"]
+xpack.security.http.ssl.verification_mode: certificate 
+xpack.security.http.ssl.key: /path/to/localhost.key
+xpack.security.http.ssl.certificate: /path/to/localhost.crt
+xpack.security.http.ssl.certificate_authorities : ["/path/to/ca.crt"]
+
+# Comment out when Kibana and Liferay's X-Pack Monitoring are also configured
+#xpack.monitoring.collection.enabled: true
 
 # For Elasticsearch 6.5/6.8
-# cluster.name: LiferayElasticsearchCluster
-# xpack.ssl.certificate: /path/to/[Elasticsearch Home]/config/localhost.crt
-# xpack.ssl.key: /path/to/[Elasticsearch Home]/config/localhost.key
-# xpack.ssl.certificate_authorities: ["/path/to/ca.crt"]
-# xpack.security.transport.ssl.enabled: true
-# xpack.security.transport.ssl.verification_mode: certificate
-# xpack.security.http.ssl.enabled: true
+#cluster.name: LiferayElasticsearchCluster
+#
+# X-Pack Security
+#xpack.security.enabled: true
+#
+# Enable TLS/SSL
+#xpack.security.transport.ssl.enabled: true # To enable Transport level SSL for internode-communication
+#xpack.security.http.ssl.enabled: true # To enable HTTP level SSL required by Kibana
+#
+## General TLS/SSL settings for both Transport and HTTP levels
+#xpack.ssl.verification_mode: certificate 
+#xpack.ssl.key: /path/to/localhost.key
+#xpack.ssl.certificate: /path/to/localhost.crt
+#xpack.ssl.certificate_authorities : ["/path/to/ca.crt"]
+#
+# Comment out when Kibana and Liferay's X-Pack Monitoring are also configured
+#xpack.monitoring.collection.enabled: true
```

Comment:
Trying to figure out if the configs we documented and those you provided are actually different is proving more time consuming than I would like. Is there a reason you overwrote all the configs instead of editing the ones that are different? Not a big deal, just wondering.

Reply:
Actually, I didn't find any that are different?

Reply:
It's organized differently.